### DISCUSSION TITLE
Exception: Indexing.NuGetPackagesPath

### DISCUSSION THREAD

**Discussion by OlivierMetrohm** [ravendb/ravendb#14194] [Q&A] [answered]
Hello there,

In our WPF application we recently switched over to a RavenDB Embedded database (5.3.102). During testing we thought to have everything working, however after performing an installation of our application, we got some new errors. 

The install location in Windows, /Program Files/, doesn't give RavenDB the access it needs to perform certain checks. Through the ServerOptions we set the LogsPath to a separate logging folder, but we keep an exception; see below. It appears to be related to the 'Additional assemblies' feature, which is something we don't intent to use, but I can't seem to find a way to prevent the startup of the embedded server from checking whether this (empty) folder checks for directory permissions.

Is there a way to disable this check, or is there an alternative solution to prevent this exception from occurring? We'd prefer to keep the RavenDBServer together with our executable after installation, but we're exploring the option to move the RavenDBServer to the ProgramData folder, where we also store the database files.

Any help would be greatly appreciated! :)

---
EXCEPTION: System.InvalidOperationException: Could not access some of the specified paths. Please check if you have sufficient privileges to access following paths:
Key: 'Indexing.NuGetPackagesPath' Path: 'C:\Program Files\<company>\<wpf product>\RavenDBServer\Packages\NuGet$1' Error: 'Access to the path 'C:\Program Files\<company>\<wpf product>\RavenDBServer\Packages\NuGet$1' is denied.'

   at Raven.Server.Config.RavenConfiguration.CheckDirectoryPermissions() in C:\Builds\RavenDB-Stable-5.3\53016\src\Raven.Server\Config\RavenConfiguration.cs:line 490

**Comment by ayende** [comment, marked as answer]
The easiest way to handle this is to set `Indexing.NuGetPackagesPath` to the `ProgramData` directory, no?

**Reply by OlivierMetrohm** [reply]
Thanks, it was indeed as simple as that... After adding a command line arg to set the path to the `ProgramData` it would start without problems.

I missed the option in the [indexing configuration](https://ravendb.net/docs/article-page/5.3/csharp/server/configuration/indexing-configuration), so didn't realize it could be set with the command line args. Thanks a lot for the fast response! 👍
